UI/Marker/EnemyPlacemark.gd

```diff
@@ -27,9 +29,12 @@ func _update_placemark_visibility(index):
 			hide()
 
 func _on_EnemyPlacemark_pressed():
-	# Left Click
-	_detailsPanel.show_details_of(enemy)
-	
+	if Input.is_key_pressed(KEY_SHIFT):
```

Comment:
Instead of having this code duplicated in both here and GatheringItemPlacemark, you could have it be a function of the superclass GenericPlacemark and call it from here and GatheringItemPlacemark